apps/src/templates/SchoolInfoConfirmationDialog.jsx

```diff
@@ -0,0 +1,95 @@
+import React, {Component, PropTypes} from 'react';
+import Dialog, {Body} from '@cdo/apps/templates/Dialog';
+import Button from './Button';
+import SchoolInfoInterstitial from '../lib/ui/SchoolInfoInterstitial';
+import i18n from '@cdo/locale';
+
+export const styles = {
+  button: {
+    marginTop: 30,
+    marginLeft: 290
+  }
+};
+
+class SchoolInfoConfirmationDialog extends Component {
+  static propTypes = {
+    schoolName: PropTypes.string,
+    onUpdate: PropTypes.func,
+    onConfirm: PropTypes.func,
+    isOpen: React.PropTypes.bool.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showSchoolInterstitial: false,
+      schoolName: props.schoolName
+    };
+  }
+
+  handleUpdateClick = () => {
+    this.setState({showSchoolInterstitial: true});
+  };
+
+  renderInitialContent() {
+    const {onConfirm} = this.props;
+    return (
+      <div style={styles.description}>
+        <Body style={styles.description}>
+          <div>
+            <p>
+              {i18n.schoolInfoDialogDescription()} Lincoln Elementary School?
```

Comment:
Why "Lincoln Elementary School"?

Reply:
Is there a design doc for this dialog? I wonder if breaking up the string so it says "Welcome Back" as a header with the question as the description would feel more welcoming?

Reply:
Lincoln Elementary School is a place holder for the school associated with a user.  In the follow- up PR, the school info will be fetched from the database for a signed in user.

Reply:
Will it not just be props.schoolName?

Reply:
@bethanyaconnor Yep, updated in PR# 2

Reply:
FWIW this seems like a small enough (and relevant) change it could've been rolled into this PR.